src/index.ts

```diff
@@ -1,6 +1,10 @@
 import initializeWasm from './helper';
 import { IFileData } from './types';
-import { IWasmModule } from './unpack';
+
+export interface IUnpackJSAPI {
```

Comment:
Hello @martinRenou good work!

I have a small suggestion. I would like to suggest to move IUnpackJSAPI to types.ts or IFileData interface to move from types.ts to index.ts and export both of them. What do you think?